<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1916</width>
    <height>1066</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="img_label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1280</width>
      <height>720</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="framerate_label">
    <property name="geometry">
     <rect>
      <x>1300</x>
      <y>50</y>
      <width>500</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Frame Rate: </string>
    </property>
   </widget>
   <widget class="QPushButton" name="start_pushButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>800</y>
      <width>200</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="right_arm_ext_count_label">
    <property name="geometry">
     <rect>
      <x>1300</x>
      <y>280</y>
      <width>600</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Right Arm Extensions:</string>
    </property>
   </widget>
   <widget class="QLabel" name="sit_to_stand_count_label">
    <property name="geometry">
     <rect>
      <x>1300</x>
      <y>210</y>
      <width>600</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Sit to Stand:</string>
    </property>
   </widget>
   <widget class="QLabel" name="left_arm_ext_count_label">
    <property name="geometry">
     <rect>
      <x>1300</x>
      <y>350</y>
      <width>600</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Left Arm Extensions:</string>
    </property>
   </widget>
   <widget class="QLabel" name="sessiontime_label">
    <property name="geometry">
     <rect>
      <x>1300</x>
      <y>100</y>
      <width>500</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Session Time:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pause_pushButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>800</y>
      <width>200</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="name_id_lineEdit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>725</y>
      <width>400</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="placeholderText">
     <string>Patient Name or ID</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1916</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionWebcam"/>
    <addaction name="actionGenerate_CSV_File"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionWebcam">
   <property name="text">
    <string>Webcam</string>
   </property>
  </action>
  <action name="actionGenerate_CSV_File">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Generate CSV File</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
